src/clang_complete.cc

```diff
@@ -149,6 +150,103 @@ lsCompletionItemKind GetCompletionKind(CXCursorKind cursor_kind) {
   }
 }
 
+void BuildCompletionItemTexts(std::vector<lsCompletionItem>& out,
+                              CXCompletionString completion_string,
+                              bool include_snippets) {
+  assert(!out.empty());
+  auto out_first = out.size() - 1;
+
+  std::string result_type;
+
+  int num_chunks = clang_getNumCompletionChunks(completion_string);
+  for (int i = 0; i < num_chunks; ++i) {
+    CXCompletionChunkKind kind =
+      clang_getCompletionChunkKind(completion_string, i);
+
+    std::string text;
+    switch (kind) {
+    case CXCompletionChunk_LeftParen:       text = '(';  break;
```

Comment:
line 168 contains
```c++
    case CXCompletionChunk_LeftParen:       text = '(';  break;
    case CXCompletionChunk_RightParen:      text = ')';  break;
```
Can they be less duplicated

Reply:
Not sure to understand, what do you mean?

Reply:
There is another chunk of code below where these character branches (`case CXCompletionChunk_LeftParen`) and string literals `text = '('` are duplicated